account_move_name_sequence/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .post_install import create_journal_sequences
+from .hooks import create_journal_sequences
```

Comment:
`from .hooks import post_init_hook`

And from that method, call the one being imported. That because a hook can run seveal methods.

This change should target 16.0